src/index.js

```diff
@@ -749,6 +749,8 @@ if (!('attachShadow' in document.createElement('div'))) {
 
     // All properties should be configurable.
     memberProperty.configurable = true;
+    // Applying to the data properties only since we can't have writable accessor properties
+    !(memberProperty.hasOwnProperty('get') || memberProperty.hasOwnProperty('set')) && (memberProperty.writable = true);
```

Comment:
It might be a more robust check to do something like:

```js
if (memberProperty.hasOwnProperty('value')) {
  memberProperty.writable = true
}
```

Or something. I think the only one's we care about making writable are the ones we specify a hard value for. The ones we don't want to be writable we're specifying using the `staticProp()` function that uses `Object.defineProperty()`.

What do you think?